src/app/news/news.component.html

```diff
@@ -1 +1,16 @@
-<p>news works!</p>
+<div class="container">
+  <div class="card" *ngFor="let card of news">
+    <div class="thumbnailImage">
+      <img [src]="imagesCover" alt="a dummy image" >
+    </div>
+    <div class="thumbnailContent">
```

Comment:
There is a convention that classes should be in lower case with dashes, like thumbnail-content, it is not wrong but it is better on the eyes. Adding it here, but applies to all the same classes.